pkg/metrics/metrics.scss

```diff
@@ -12,6 +12,17 @@
   }
 }
 
+html {
+  --metrics-heading-color: #fff;
+  --metrics-heading-time-color: #eee;
+}
+
+html.pf-theme-dark {
+  --metrics-heading-color: var(--pf-global--BackgroundColor-dark-100);
+  --metrics-heading-time-color: var(--pf-global--BackgroundColor--300);
+  --metrics-events-border-color: #000;
+}
+
```

Comment:
@garrett Please check if you like this approach or if we should just use a fallback.

Reply:
It's probably more clear to redefine it, like here, rather than use a fallback. This way, we can keep the definition the same in the CSS later on and just swap out the meaning. And we can keep the variables and their values together in the same place in the CSS.